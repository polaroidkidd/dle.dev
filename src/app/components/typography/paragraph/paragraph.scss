@import "src/app/styles/main";


.paragraph {
  color: $color-text-primary;
  font-family: Poppins, sans-serif;
  font-size: 2rem;
  line-height: 3rem;
  font-weight: 200;
  letter-spacing: 0.75rem;
  @media (max-width: 768px) {
    font-size: 1rem;
    font-weight: 200;
  }
}