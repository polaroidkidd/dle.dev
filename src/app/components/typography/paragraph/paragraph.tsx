import React from "react";
import "./paragraph.scss";

type Props = {
  text: string
}

const Paragraph: React.FC<Props> = ({text}) => {
  return (
    <p className="paragraph">{text}</p>
  );
};
export { Paragraph };