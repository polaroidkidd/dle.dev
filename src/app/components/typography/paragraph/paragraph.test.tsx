import React from "react";
import renderer from "react-test-renderer";
import { Paragraph } from "./paragraph";


test("Render Paragraph with text", () => {
  const tree = renderer.create(<Paragraph text="test heading"/>).toJSON();
  expect(tree).toMatchSnapshot();
});

test("Render Paragraph with empty string", () => {
  const tree = renderer.create(<Paragraph text=""/>).toJSON();
  expect(tree).toMatchSnapshot();
});
