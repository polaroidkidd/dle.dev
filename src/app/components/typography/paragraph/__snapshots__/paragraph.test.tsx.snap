// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Paragraph with empty string 1`] = `
<p
  className="paragraph"
>
  
</p>
`;

exports[`Render Paragraph with text 1`] = `
<p
  className="paragraph"
>
  test heading
</p>
`;
